.github/workflows/python-packaging.yml

```diff
@@ -102,11 +102,6 @@ jobs:
           only: cp${{ matrix.python-version }}-${{ matrix.cibw_build }}
         env:
           CIBW_BUILD_VERBOSITY: 1
-          # Delete lib folder that apis/python/setup.py:find_or_build() looks for in deciding to
-          # run CMake build or not. Otherwise it'll keep reusing the library file built in the
-          # first iteration of cibuildwheel's outer loop, resulting in wheels with the library
-          # built for the wrong python version.
```

Comment:
This seems like a useful comment. Is it not still useful? cc @jdblischak .

Reply:
ICYMI, it's a copy of the same comment we discussed in [the other thread](https://github.com/single-cell-data/TileDB-SOMA/pull/3909/files#r2027765704).

In my reply there, I reasoned that neither copy was necessary; this view is of the subsequent commit I made, removing the remaining copy.

Reply:
> This seems like a useful comment. Is it not still useful? cc @jdblischak .

I'm not familiar with that comment nor this workflow in general, so I am unsure of its usefulness. I defer to Ryan's opinion.

From a quick `git blame` investigation, I believe this comment was included in the original PR https://github.com/single-cell-data/TileDB-SOMA/pull/605 back in Dec 2022 (https://github.com/single-cell-data/TileDB-SOMA/commit/aef8058ef6e14781c56bf51f23faca0b4cd8c2d5)